relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -60,6 +71,22 @@ pub(crate) fn scrub_span_description(
             ("http", _) => scrub_http(description, span_allowed_hosts),
             ("cache", _) | ("db", "redis") => scrub_redis_keys(description),
             ("db", _) if db_system == Some("redis") => scrub_redis_keys(description),
+            ("db", _)
+                if scrub_mongo_description == ScrubMongoDescription::Enabled
+                    && db_system == Some("mongodb") =>
+            {
+                let command = data
+                    .and_then(|data| data.db_operation.value())
+                    .and_then(|command| command.as_str());
+
+                let collection = data
+                    .and_then(|data| data.db_collection_name.value())
```

Comment:
With SQL, we learned the hard way that collections / tables can contain all kinds of identifiers (see [test cases here](https://github.com/getsentry/relay/blob/b0ddc97eddcf0e3ed173620668e5cb6110ddf422/relay-event-normalization/src/normalize/span/description/sql/mod.rs#L911-L957)). IMO we should pass the mongodb collection name through the same scrubbing as SQL table names to catch those identifers, i.e. apply the [`TABLE_NAME_REGEX`](https://github.com/getsentry/relay/blob/b0ddc97eddcf0e3ed173620668e5cb6110ddf422/relay-event-normalization/src/normalize/span/description/sql/parser.rs#L23-L24).